packages/venia-ui/lib/components/CartPage/ProductListing/product.css

```diff
@@ -85,16 +84,21 @@
     grid-area: stock;
 }
 
+.stockStatusMessage:empty {
+    display: none;
+}
+
 @media (max-width: 640px) {
     .details {
-        grid-template-areas:
-            'name'
-            'options'
-            'price'
-            'stock'
-            'quantity';
-        grid-template-columns: auto;
-        grid-template-rows: auto;
+        grid-template: none;
```

Comment:
This resets `grid-template-areas`, `grid-template-columns`, and `grid-template-rows` in one line. As a result, each grid item creates a new row (when its `grid-area` isn't set).